src/DsaPrinter.cc

```diff
@@ -542,6 +544,32 @@ struct DOTGraphTraits<sea_dsa::Graph *> : public DefaultDOTGraphTraits {
       }
     }
 
+    // print edges from call sites to cells
+    //TODO: BUG: the callsite is drawn twice. One as a callsite and one as a value
```

Comment:
That's not a bug. A call of the form `x = (*y)(a)`, will appear twice. Once as a call site, and once as `x`, if `x` is a pointer.
These are two difference objects. Object `x` refers to the return of the call, and call-site to the yet unresolved call.